test/models/orchestration/tftp_test.rb

```diff
@@ -230,6 +230,37 @@ class TFTPOrchestrationTest < ActiveSupport::TestCase
     end
   end
 
+  test "generate_default_pxe_template_for_pxelinux_localboot_from_setting" do
+    if unattended?
```

Comment:
Use an early return, `return unless unattended?` to avoid nesting the whole test, feel free to change other tests in this file